gui/projectfile.h

```diff
@@ -107,6 +107,21 @@ class ProjectFile : public QObject {
         mCheckUnusedTemplates = b;
     }
 
+    /**
+     * @brief Get inline suppression usage.
```

Comment:
I am thinking that the comment could be a bit misleading however I don't know if it matters a lot. It's quite easy to understand what the flag does.